shared/templates/mount_option_home/oval.template

```diff
@@ -0,0 +1,77 @@
+        <ind:pattern operation="pattern match" var_ref="var_home_partitions_mount_points_{{{ MOUNTOPTION }}}_regex"
+        var_check="at least one"/>
+        <ind:instance datatype="int">1</ind:instance>
+    </ind:textfilecontent54_object>
+
+    <linux:partition_object id="object_home_partition_all_partitions_{{{ MOUNTOPTION }}}" version="1">
+        <linux:mount_point operation="not equal">/</linux:mount_point>
+    </linux:partition_object>
+
+    <local_variable id="var_home_partitions_mount_points_{{{ MOUNTOPTION }}}_regex" datatype="string" version="1"
+    comment="Variable including all home dirs from interactive users">
+        <concat>
+            <literal_component>^(?:(?!nobody).*)?:.*?:[1-9]\d{3,}:.*?:.*?:(</literal_component>
+            <object_component item_field="mount_point"
+            object_ref="object_home_partition_all_partitions_{{{ MOUNTOPTION }}}" />
+            <literal_component>).*?:.*</literal_component>
+        </concat>
+    </local_variable>
+
+    <local_variable id="var_home_partitions_mount_points_{{{ MOUNTOPTION }}}" datatype="string" version="1"
+    comment="This variable collects all partition mount points for interactive users">
+        <object_component item_field="subexpression"
+        object_ref="object_home_partitions_mount_points_{{{ MOUNTOPTION }}}" />
+    </local_variable>
+
+    <linux:partition_test check="all" check_existence="all_exist"
+    id="test_home_partition_{{{ MOUNTOPTION }}}" version="1" comment="{{{ MOUNTOPTION }}} on Home mount points">
+        <linux:object object_ref="object_home_partitions_{{{ MOUNTOPTION }}}" />
+        <linux:state state_ref="state_home_partition_{{{ MOUNTOPTION }}}" />
+    </linux:partition_test>
+    <linux:partition_object id="object_home_partitions_{{{ MOUNTOPTION }}}" version="1">
+        <linux:mount_point operation="equals" var_ref="var_home_partitions_mount_points_{{{ MOUNTOPTION }}}"
+        var_check="at least one"/>
+    </linux:partition_object>
+    <linux:partition_state id="state_home_partition_{{{ MOUNTOPTION }}}" version="1">
+        <linux:mount_options datatype="string" entity_check="at least one" operation="equals">{{{ MOUNTOPTION }}}</linux:mount_options>
+    </linux:partition_state>
+
+    <ind:variable_test check="none satisfy" check_existence="all_exist"
+    comment="Home mount points aren't in unexpected partition"
+    id="test_home_allowed_partition_{{{ MOUNTOPTION }}}" version="1">
+      <ind:object object_ref="object_home_allowed_partition_{{{ MOUNTOPTION }}}" />
+      <ind:state state_ref="state_home_allowed_partition_{{{ MOUNTOPTION }}}" />
+    </ind:variable_test>
+    <ind:variable_object id="object_home_allowed_partition_{{{ MOUNTOPTION }}}" version="1">
+      <ind:var_ref>var_home_partitions_mount_points_{{{ MOUNTOPTION }}}</ind:var_ref>
+    </ind:variable_object>
+    <ind:variable_state comment="unexpected partition"
+    id="state_home_allowed_partition_{{{ MOUNTOPTION }}}" version="1">
```

Comment:
We should keep the indentation aligned to other templates and the line sizes aligned to the [style guide](https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html).